### PR TITLE
CM-113: add Group to django orm

### PR DESCRIPTION
**Pull request by jdolkowski**
https://openimis.atlassian.net/browse/CM-113

I was using ContributionPlanBundleDetails as a reference. It allows to add one pair of group and individual. Wouldn’t it be a good idea to allow multiple indvidual ids?

### CODE REVIEW

File: social_protection/services.py [hunk 1]
```diff
@@ -4,12 +4,21 @@
...
     BenefitPlan,
-    Beneficiary
+    Beneficiary, Group, GroupIndividual
 )
 from social_protection.validation import (
     BeneficiaryValidation,
-    BenefitPlanValidation
+    BenefitPlanValidation, GroupValidation, GroupIndividualValidation
 )
+from abc import ABC
```
Comment (sniedzielski) [inline social_protection/services.py:13]: Please fix the order of imports. Keep in mind such suggestions of order: 
- Start with standard library imports: Import Python's built-in modules first followed by any third-party library imports
-  import Django modules: Import Django modules in a separate block, below the standard library imports.
- Import local project-specific modules: Import your project's modules last. These can be files or packages specific to your Django project. 

Moreover please separate imports with blank lines: Use a blank line to separate different import groups (standard library, Django, project-specific). This improves visual clarity and makes it easier to distinguish between different import sections.

File: social_protection/tests/data.py [hunk 2]
```diff
@@ -91,3 +91,15 @@
     "date_valid_from": "2023-01-01",
     "date_valid_to": "2023-12-31",
 }
+
+service_group_add_payload = {
+    "status": "Potential",
```
Comment (sniedzielski) [inline social_protection/tests/data.py:96]: I think now status is not 'Potential' but 'POTENTIAL' based on enum value and your previous approaches

File: social_protection/tests/data.py [hunk 3]
```diff
@@ -91,3 +91,15 @@
...
+    "date_valid_from": "2023-01-01",
+    "date_valid_to": "2023-12-31",
+}
+
+service_group_update_payload = {
+    "status": "Active",
```
Comment (sniedzielski) [inline social_protection/tests/data.py:102]: I think now status is not 'Active' but 'ACTIVE' based on enum value and your previous approaches

**Review by sniedzielski — COMMENTED** [review]
Generally LGTM, There is nothing critical according to me.  I left one comment regarding imports statement in python/Django and two minor ones referred to the test data. 

Let's wait for another developer's feedback too

File: social_protection/models.py [hunk 4]
```diff
@@ -20,4 +28,14 @@ class BenefitPlan(core_models.HistoryBusinessModel):
 class Beneficiary(core_models.HistoryBusinessModel):
     individual = models.ForeignKey(Individual, models.DO_NOTHING, null=False)
     benefit_plan = models.ForeignKey(BenefitPlan, models.DO_NOTHING, null=False)
-    status = models.CharField(max_length=100, null=False)
+    status = models.CharField(max_length=100, choices=StatusChoices.choices, null=False)
+
+
+class Group(core_models.HistoryBusinessModel):
```
Comment (sniedzielski) [inline social_protection/models.py:34]: I think this should be placed in 'Individual' module.

File: social_protection/models.py [hunk 5]
```diff
@@ -20,4 +28,14 @@ class BenefitPlan(core_models.HistoryBusinessModel):
...
+    benefit_plan = models.ForeignKey(BenefitPlan, models.DO_NOTHING, null=False)
+    status = models.CharField(max_length=100, choices=StatusChoices.choices, null=False)
+
+
+class GroupIndividual(core_models.HistoryBusinessModel):
```
Comment (sniedzielski) [inline social_protection/models.py:39]: I think this should be placed in 'Individual' module.

Comment (delcroip) [reply social_protection/models.py:39]: yes or group module

**Pull request closed without merging**